core/teamserver/modules/boo/uploader.py

```diff
@@ -0,0 +1,55 @@
+from core.teamserver.module import Module
+

```

Comment:
@Tinydile Tests are failing cause you're missing an import for the `print_bad` function, can you add that in when you get a sec?  :)

Thanks!